Implement support for TLS/SSL on the HTTP interface (#1040)
strimzi/strimzi-kafka-bridge: documentation/modules/con-securing-http-interface.adoc

= Securing the HTTP Bridge HTTP interface

[role="_abstract"]
By default, connections between HTTP clients and HTTP Bridge are not encrypted,
but you can configure a TLS-encrypted connection through its xref:proc-configuring-kafka-bridge-{context}[properties file].

> If the HTTP Bridge is configured with TLS encryption, client requests must use HTTPS instead of HTTP.

Reviewer: ```suggestion
If the HTTP Bridge is configured with TLS encryption, client should connect to the bridge by using HTTPS instead of HTTP.
```

= Securing the HTTP Bridge HTTP interface

[role="_abstract"]

> By default, requests sent from clients to the HTTP Bridge are sent without encryption therefore they must use HTTP.

Reviewer: I would not put the sentence this way. I mean, the statement is from a client perspective saying that by default their requests are not encrypted (but how do you know? you are not in control of clients). We should put the sentence from a bridge perspective saying that by default the connection with client is not encrypted but it can be enabled. Hopefully you got what I mean :-)
